analytics/lib/counts.py

```diff
@@ -111,6 +112,17 @@ def do_delete_count_stat_at_hour(stat, end_time):
     RealmCount.objects.filter(property = stat.property, end_time = end_time).delete()
     InstallationCount.objects.filter(property = stat.property, end_time = end_time).delete()
 
+def get_group_by_arguments(stat):
+    # type: (CountStat) -> Tuple[str, str]
+    if stat.group_by is None:
+        # if none, we compose the query with inserting the default null value into the subgroup column
+        subgroup = ', NULL'
```

Comment:
no; that doesnt work--if you look at the interpolation if the query itself includes the comma, then the case where  we do have a not null subgroup will throw an error ( there will be two commas, since the not null needs the comma as its used in the group by as well.